src/test/java/calculator/CalculatorTest.java

```diff
@@ -0,0 +1,82 @@
+package calculator;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CalculatorTest {
+    private Calculator calculator;
+
+    @BeforeEach
+    void setUp() {
+        calculator = new Calculator();
+    }
+
+    @Test
+    @DisplayName("쉼표(,) 또는 콜론(:)을 구분자로 가지는 문자열의 합을 계산한다.")
+    void defaultSeparatorTest() {
+        String input = "1,2:3,4:5,6";
+
+        int result = calculator.calculate(input);
+
+        assertEquals(21, result);
```

Comment:
`assertEquals`의 기댓값 인자와 검증 대상 인자를 올바른 순서대로 잘 사용하셨네요. 👍 
assertj 의 `assertThat`을 이용하면 인자의 순서도 헷갈리지 않고, 가독성도 좋으니 한 번 사용해 보셔도 좋겠습니다.

```suggestion
        assertThat(result).isEqualTo(21);
```

Reply:
아하 assertj가 이런 장점이 있겠군요 다음 과제부터 제대로 적용해보겠습니다 감사합니다 :) 
일부 반영했습니다 ([2d21e1b](https://github.com/next-step/java-racingcar-kakao/pull/4/commits/2d21e1b0be49a64b79c0e2432064775d77613e53))